🔴 [BUILD BLOCKER] Wallet package TypeScript errors: 'network' property doesn't exist on Chain type

## Problem
The wallet package fails to build due to TypeScript errors in `packages/wallet/src/chains.ts`.

## Error Details
```
src/chains.ts(7,3): error TS2353: Object literal may only specify known properties, and 'network' does not exist in type 'Chain'.
src/chains.ts(35,3): error TS2353: Object literal may only specify known properties, and 'network' does not exist in type 'Chain'.
```

## Impact
- **Severity**: CRITICAL - P0
- The entire project build fails (`npm run build` exits with code 2)
- Cannot deploy or run the application
- Blocks all development work

## Root Cause
The `network` property was removed from the `Chain` type in wagmi v2/viem v2. The chain configuration is using outdated syntax.

## Location
- **File**: `packages/wallet/src/chains.ts`
- **Lines**: 7, 35

## Suggested Fix
Remove the `network` property from both `baseMainnet` and `baseSepolia` chain definitions. In wagmi v2, the `name` and `id` properties are sufficient for chain identification.

## Priority
**P0 - Critical**: This must be fixed before any build or deployment can succeed.

--- a/packages/wallet/src/chains.ts
+++ b/packages/wallet/src/chains.ts
@@ -56,3 +56,48 @@ export const baseSepolia: Chain = {
 };
 
 export const supportedChains = [baseMainnet, baseSepolia] as const;
+import { Chain } from "viem";
+
+// Base Mainnet
+export const baseMainnet: Chain = {
+  id: 8453,
+  name: "Base",
+  nativeCurrency: {
+    name: "Ether",
+    symbol: "ETH",
+    decimals: 18,
+  },
+  rpcUrls: {
+    default: {
+      http: ["https://mainnet.base.org"],
+    },
+  },
+  blockExplorers: {
+    default: {
+      name: "BaseScan",
+      url: "https://basescan.org",
+    },
+  },
+};
+
+// Base Sepolia Testnet
+export const baseSepolia: Chain = {
+  id: 84532,
+  name: "Base Sepolia",
+  nativeCurrency: {
+    name: "Ether",
+    symbol: "ETH",
+    decimals: 18,
+  },
+  rpcUrls: {
+    default: {
+      http: ["https://sepolia.base.org"],
+    },
+  },
+  blockExplorers: {
+    default: {
+      name: "BaseScan",
+      url: "https://sepolia.basescan.org",
+    },
+  },
+};

